indico/modules/events/tracks/models/tracks.py

```diff
@@ -140,29 +126,27 @@ def locator(self):
     def __repr__(self):
         return format_repr(self, 'id', _text=text_to_repr(self.title))
 
+    def can_access(self, user):
+        # not relevant to tracks
+        raise NotImplementedError
+
     def can_delete(self, user):
         return self.event.can_manage(user) and not self.abstracts_accepted
 
-    def can_review_abstracts(self, user):
+    def can_review_abstracts(self, user, allow_admin=True):
```

Comment:
`allow_admin` isn't used anywhere and `explicit_permissions` makes it ineffective anyway